### ISSUE TITLE
Unable to reload the test with old test runner

### ISSUE THREAD

**Issue by jakub-g** [attester/attester#90]
When one starts the campaign with attester, it registers the tests to be run internally.

There's a link on attester status page to run the old tests as a way to facilitate debugging. It launches `interactive.html` page which injects a generated fake test suite as a MainTestSuite so that the old runner can work properly:

The fake MainTestSuite looks like that:

```
Aria.classDefinition({$classpath:"MainTestSuite",$extends:"aria.jsunit.TestSuite",$constructor:function(){this.$TestSuite.constructor.call(this);
this.addTests("test.aria.core.BrowserTest");
}});
//# sourceURL=/campaign1/MainTestSuite.js 
```

And the code responsible for making it available:

```
attester.currentTask.start = function () {
    var width = {
        min: 180
    };
    var height = {
        min: 342
    };
    aria.core.DownloadMgr.updateRootMap({
        "MainTestSuite": "./"
    });
    aria.core.DownloadMgr.updateUrlMap({
        "MainTestSuite": "MainTestSuite.js"
    }); 
    ...
```

It's all fine until the user clicks "reload the suite" button; then somehow the root map is cleaned and MainTestSuite is requested from another place, 404ing (you can see in the net panel) and raising an error:

![__old runner](https://cloud.githubusercontent.com/assets/1437027/3104900/1742237c-e66d-11e3-93b9-6c7cb8ed9bc3.png)

To sum up, we should find a way to reestablish rootmap after reloading the suite.
